### PR TITLE
IGAPP-1340: Adjust colors and icons

### PR DESCRIPTION
**Pull request by sarahsporck**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

@f1sh1918 I get an issue with an icon that cannot be fetched from the cms when i start the dev server. Can you look into this until I can log into the cms myself? :)

Testable with `yarn start:obdach`

### CODE REVIEW

**Comment by f1sh1918** [comment]
> This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/). You can find it there by looking for an issue with the key which is mentioned in the title of this pull request. It starts with the keyword **IGAPP**.
> 
> @f1sh1918 I get an issue with an icon that cannot be fetched from the cms when i start the dev server. Can you look into this until I can log into the cms myself? :)
> 
> Testable with `yarn start:obdach`

I guess this relies to some new imageCache maybe. You can activate CORS for local development in this case @sarahsporck

File: web/src/components/HeaderLogo.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ const LogoContainer = styled.div`
   display: flex;
   justify-content: start;
   align-items: center;
-  flex: 0 1 220px;
+  flex: initial;
```
Comment (f1sh1918) [inline web/src/components/HeaderLogo.tsx:21]: tbh couldn't see that this is needed! I think you can remove it

**Review by f1sh1918 — APPROVED** [review]
Works fine. just one small comment

**Review by LeandraH — APPROVED** [review]
Looks good, tested in Firefox

**Review by steffenkleinle — APPROVED** [review]

File: web/src/components/HeaderLogo.tsx (hunk 1 above)
Comment (sarahsporck) [reply web/src/components/HeaderLogo.tsx:21]: Without this it looks like this for me on firefox: 
Bildschirmfoto 2023-07-27 um 10 35 52
And with `flex: initial` it looks like this:
Bildschirmfoto 2023-07-27 um 10 35 52

**Pull request merged**